UsefulFlags.md

```diff
@@ -62,10 +62,14 @@ bazel test //foo:foo_tests --apple_platform_type=macos
 
 Select the version of Xcode to use. Bazel auto-discovers installed versions of Xcode. If you've recently installed a version of Xcode that Bazel can't find with `--xcode_version`, you might have to restart the daemon by running `bazel clean --expunge`.
 
-###### `--objc_enable_binary_stripping=true`
+###### `--objc_enable_binary_stripping=true` and `--features=dead_strip`
 
-Strips dead symbols from `objc_binary` outputs. This flag only takes effect if you are building your application with `--compilation_mode=opt` and is a link-time optimization. It tells bazel to pass the `-dead-strip` flag at link time. If you are statically linking your entire application this can greatly reduce your binary size.
+Strips unreachable functions and data from the final binary. These flags tell bazel to pass the `-dead-strip` flag at link time. If you are statically linking your entire application this can greatly reduce your binary size.
 
+* For `objc_binary`, use `--objc_enable_binary_stripping=true`
```

Comment:
@rahul-malik I took `objc_binary` from your original copy, but should this be mentioning the rules_apple application rules?

Reply:
@kastiglione - They probably should since it seems like the `*_binary` rules are really just implementation details of a higher-level rule like `command_line_application`/`ios_application`/`macos_application`/...